modules/nodes-vsphere-machine-set-scaling-static-ip.adoc

```diff
@@ -0,0 +1,193 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/node-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-vsphere-machine-set-scaling-static-ip_{context}"]
+= Using a machine set to scale machines with configured static IP addresses
+
+You can use a machine set to scale machines with configured static IP addresses. 
+
+:FeatureName: Static IP addresses for vSphere nodes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+After you configure a machine set to request a static IP address for a machine, the machine controller creates an `IPAddressClaim` resource in the `openshift-machine-api` namespace. The external controller then creates an `IPAddress` resource and binds any static IP addresses to the `IPAddressClaim` resource.
+
+[IMPORTANT]
+====
+Your organization might use numerous types of IP address management (IPAM) services. If you want to enable a particular IPAM service on {product-title}, you might need to manually create the `IPAddressClaim` resource in a YAML definition and then bind a static IP address to this resource by entering the following command in your `oc` CLI:
+
+[source, terminal]
+----
+oc create -f <ipaddressclaim_filename>
+----
+
+See https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/configuring-additional-network.html#nw-multus-ipam-object_configuring-additional-network 
```

Comment:
SME query: Worthwhile linking to this section? 

https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/configuring-additional-network.html#nw-multus-ipam-object_configuring-additional-network

Reply:
While IPAM is used in both scenarios, how it is used and implemented is quite different.  I don't think we should link to this document.

Reply:
Thanks. Removed the link.

Reply:
Removed link.